ui/submodules/packages/components/src/components/cellRenderers/nameCell.tsx

```diff
@@ -7,12 +7,16 @@ import { OpenInNew } from "@mui/icons-material";
 import { scenarioHref } from "./categoriesCell";
 import { useFilterContext } from "../filters/filtersContext";
 
+export function IconImg({ title, src }: { title?: string; src: string }): JSX.Element {
+    return <img title={title} style={{ height: "1.5em", verticalAlign: "middle" }} src={src} />;
```

Comment:
Icon url has the same format like others icons, it can be relative path or url (with https/http)

Reply:
so what?
and relative to what?